python/ray/ml/predictors/integrations/tensorflow/tensorflow_predictor.py

```diff
@@ -121,4 +128,23 @@ def build_model(self):
         Returns:
             DataBatchType: Prediction result.
         """
-        raise NotImplementedError
+        data = self.preprocessor.transform_batch(data)
```

Comment:
Same comment for this one- let's also support the case when `self.preprocessor` is None. Thanks!